apps/fz_http/assets/js/wg_conf.js

```diff
@@ -15,7 +15,10 @@ const renderConfig = function () {
     const privateKey = sessionStorage.getItem(publicKey)
 
     // XXX: Clear all private keys
-    sessionStorage.removeItem(publicKey)
+    setTimeout(() => {
+      sessionStorage.removeItem(publicKey)
+    }, 1000 * 60 * 5);
```

Comment:
`send_update` triggers a re-render of the component, which causes the hook to be run twice. The first time clears the private key and the second time has nothing to work with and will error.

Keeping the keys around for a while. Session storage is cleared anyway when the tab is closed. Not sure how big of a risk this causes?

Reply:
A few seconds is sufficient. The 2 renders are back to back.